samples/bluetooth/bap_broadcast_source/prj.conf

```diff
@@ -14,3 +14,5 @@ CONFIG_BT_ISO_TX_BUF_COUNT=6
 CONFIG_BT_ISO_TX_MTU=60
 
 CONFIG_BT_DEVICE_NAME="Broadcast Audio Source"
+CONFIG_STATIC_BROADCAST_ID=y
+CONFIG_BROADCAST_ID=0x123456
```

Comment:
It's weird to define a Kconfig that defaults to `n` and then enable it by default. 

Suggest to either use random by default (i.e. remove these lines), or make `STATIC_BROADCAST_ID` default to `y`

Reply:
I agree with this. However, `CONFIG_BROADCAST_ID=0x123456` still needs to be there

Reply:
Indeed :)